### PR TITLE
Staging v5.0.4

### PR DESCRIPTION
**Pull request by Kevingislason**
## Description

Release v5.0.4

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [x] Re-reviewed files changed in the Github PR explorer

## If PR to `main`
- [x] Update the version number in `package.json` and `src/config/common.json`
- [x] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [x] Update `CHANGELOG.md`
- [x] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

**Comment by Kevingislason** [comment]
Reverted #750 because it caused some strange CSS issue where the button in the attached photo would appear without its normal gradient background. We can revisit this bit later.
Screen Shot 2023-03-22 at 8 21 13 PM
Screen Shot 2023-03-22 at 8 20 16 PM

**Review by jincan39 — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -224,12 +224,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [5.0.4] - 2023-3-23
 
 ### Fixed
-- [\#753](https://github.com/Manta-Network/manta-front-end/pull/753) Fix a single disconnected network on Bridge Page from preventing all transactions
+- [\#753](Branch) Fix a single disconnected network on Bridge Page from preventing all transactions
```
Comment (jasonintju) [inline CHANGELOG.md:227]: Just curious, what does 'Branch' mean here? If there isn't a link here, click '#753' won't work.

**Review by jasonintju — DISMISSED** [review]

**Review by jincan39 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -214,4 +214,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [5.0.3] - 2023-3-15
 
 ### Fixed
-- [\#755](https://github.com/Manta-Network/manta-front-end/pull/755) Fix dotmenu content height style
\ No newline at end of file
+- [\#755](https://github.com/Manta-Network/manta-front-end/pull/755) Fix dotmenu content height style
+
+## [5.0.3] - 2023-3-15
```
Comment (jasonintju) [inline CHANGELOG.md:219]: Repeated [5.0.3]

**Review by jasonintju — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Kevingislason) [reply CHANGELOG.md]: nice catch, thanks

**Review by jincan39 — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -219,7 +219,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [5.0.4] - 2023-3-28
 
 ### Fixed
-- [\#829](https://github.com/Manta-Network/manta-front-end/pull/829), [\#753](https://github.com/Manta-Network/manta-front-end/pull/732)  Fix a single disconnected network on Bridge Page from preventing all transactions
+- [\#829](https://github.com/Manta-Network/manta-front-end/pull/829) Fix a single disconnected network on Bridge Page from preventing all transactions
+- [\#753](https://github.com/Manta-Network/manta-front-end/pull/732) Fix a single disconnected network on Bridge Page from preventing all transactions
```
Comment (jasonintju) [inline CHANGELOG.md:223]: take it easy, the link should be 753

**Review by jincan39 — APPROVED** [review]
should we put #829 and #753 in same line if they fixed the same issue

**Comment by Kevingislason** [comment]
I don't know what the proper convention is tbh

**Pull request merged**